Robust_Lab.qmd

```diff
@@ -112,8 +377,15 @@ e\. Compare the observed mean difference to the NHD (is *p* \< .05?)
 
     a\. Run a permutation test (ANOVA)
 
+    ```{r}
+    mod <- aovperm(formula= errors~convo*drive, data = fac_data, np=10000)
+    mod
+    ```
+
     b\. How would you follow-up significant effects in this context?
 
+Although the interaction effect is marginal (p = 0.07), you could possibly follow it up with a simple slop analysis. In addition you could conduct a simple effects analysis to look at differences between errors for conversation levels at a given level of driving difficulty (using the emmeans package).
+
```

Comment:
Here I meant any significant effects (main and interaction). If I were to follow up with posthoc tests here, I would  bootstrap the pairwise differences for each comparison to get mean difference and 95% CI.